packages/opentelemetry-sdk-trace-base/src/BasicTracerProvider.ts

```diff
@@ -69,8 +69,7 @@ export class BasicTracerProvider implements TracerProvider {
   private readonly _config: TracerConfig;
   private readonly _tracers: Map<string, Tracer> = new Map();
   private readonly _resource: IResource;
-
-  private activeSpanProcessor: MultiSpanProcessor;
+  private readonly _activeSpanProcessor: MultiSpanProcessor;
```

Comment:
note for reviewer: also made it `readonly` because is not meant to be changed once is set in the constructor